app/assets/javascripts/controllers/container_dashboard/charts/heatmap/heatmap-legend-directive.js

```diff
@@ -0,0 +1,32 @@
+angular.module('miq.charts').directive('heatMapLegend',
+  function() {
```

Comment:
This directive will also be available in Angular Patternfly 2.6 which releases tomorrow (11/10/15).